src/components/Chat/Chat.vue

```diff
@@ -74,11 +74,21 @@
         <chat-toolbar :partner-id="partnerId">
           <template #avatar-toolbar>
             <ChatAvatar
+              v-if="enabledNodes"
               class="chat-avatar"
               :user-id="partnerId"
               use-public-key
               @click="onClickAvatar(partnerId)"
             />
+            <v-progress-circular
+              v-else
+              :class="`mr-3`"
```

Comment:
There is no need in string literal here

```suggestion
              class="mr-3"
```